app/src/main/res/layout/activity_log_in.xml

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <ImageButton
+            android:id="@+id/backLogin"
+            android:layout_width="48dp"
+            android:layout_height="48dp"
+            android:layout_margin="@dimen/space20"
+            android:background="@null"
+            android:contentDescription="@string/desc_img"
+            android:src="@drawable/arrow_back"
+            app:layout_constraintBottom_toTopOf="@id/hello_message"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <TextView
+            android:id="@+id/hello_message"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:padding="@dimen/space20"
+            android:text="@string/hello_message"
+            android:textSize="34sp"
+            android:textStyle="bold"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@+id/backLogin" />
+
+        <TextView
+            android:id="@+id/signin_text"
+            android:layout_width="match_parent"
```

Comment:
Please don't use match parent inside ConstraintLayout, refer to [documenation](https://developer.android.com/reference/androidx/constraintlayout/widget/ConstraintLayout)